.gitignore

```diff
@@ -7,4 +7,11 @@ CMakeFiles/
 *.cxx_parameters
 lib*.so.*
 Makefile
+src/stable/interfaces/cpp/
+src/stable/interfaces/java/
 src/stable/interfaces/python/
+
+src/stable/components/basic_server/myInterface.h
```

Comment:
Why these files must be ignored?
They are self generated?
(Ok, generated by slice).
To avoid these changes at global gitignore, you can use out-of-source build and send slice output to build directory (a include_directories may be mandatory)